pyppeteer/timeout_settings.py

```diff
@@ -15,22 +16,22 @@ def __init__(self):
         self._defaultTimeout = None
         self._defaultNavigationTimeout = None
 
-    def setDefaultTimeout(self, timeout):
+    def setDefaultTimeout(self, timeout: Union[float, int]):
```

Comment:
Maybe a `Number` type would be nice: `Number = Union[float, int]`

Reply:
Good question. Apparently `float` should be enough as per this explanation: https://stackoverflow.com/questions/50928592/mypy-type-hint-unionfloat-int-is-there-a-number-type